migration-engine/connectors/migration-connector/src/migrations_directory.rs

```diff
@@ -64,6 +64,7 @@ pub fn write_migration_lock_file(migrations_directory_path: &str, provider: &str
     let mut file = std::fs::File::create(&file_path)?;
     let content = format!(
         r##"# Please do not edit this file manually
+# It should be added in your version-control system (i.e. Git)
```

Comment:
super nitpicky, but "e.g." would be more accurate than "i.e" there.

Reply:
Oh right

Reply:
I guess it's ok like this 😄